build-parent/pom.xml

```diff
@@ -82,10 +82,9 @@
     <version.zipkin>1.13.1</version.zipkin>
 
     <!-- OpenTracing related versions -->
-    <version.opentracing.tracerresolver>0.1.0</version.opentracing.tracerresolver>
-    <version.opentracing.servlet>0.0.9</version.opentracing.servlet>
-    <version.opentracing.spanmanager>0.0.5</version.opentracing.spanmanager>
-    <version.jaeger>0.20.0</version.jaeger>
+    <version.opentracing.tracerresolver>0.1.4</version.opentracing.tracerresolver>
+    <version.opentracing.servlet>0.1.0</version.opentracing.servlet>
+    <version.jaeger>0.24.1-SNAPSHOT</version.jaeger>
```

Comment:
Did you mean to use a SNAPSHOT version?

We can't release with a SNAPSHOT

Reply:
> This is currently WIP, pending release of Jaeger Client Java (jaegertracing/jaeger-client-java#365))

:)